src/battle-animations.ts

```diff
@@ -106,6 +107,7 @@ class BattleScene {
 		let numericId = 0;
 		if (battle.id) {
 			numericId = parseInt(battle.id.slice(battle.id.lastIndexOf('-') + 1), 10);
+			if (this.battle.id.includes('digimon')) this.digi = true;
```

Comment:
See below - I think you can make this more generic.

Reply:
I see, Slayer or Fender set up that code, it's very stable so we do not wish to change it, for example when our test client fell apart that still worked perfectly. If you would like to change it you have my permission to do so.

Reply:
This change is all of 9 lines, it really should be trivial at this stage to change and I would be very concerned if you were unable to at this point.

I wouldn't merge this as is, but maybe Zarel won't care because he just wants to be done with this.

Reply:
your more then welcome to change it but we are not doing it, we want to be done just as much as zarel.